tests/test_tool.py

```diff
@@ -6,7 +6,7 @@
     plt = None
 
 import rasterio
-from rasterio.tool import show, show_hist, stats
+from rasterio.rio.insp import show, show_hist, stats
```

Comment:
With your modification in response to my comment, this import also needs to be modified.